src/main/java/ai/shreds/domain/DomainSupplierEntity.java

```diff
@@ -0,0 +1,65 @@
+package ai.shreds.domain;
+
+import ai.shreds.shared.SharedSupplierDTO;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.PrePersist;
+import javax.persistence.PreUpdate;
+import javax.persistence.Table;
+import javax.validation.constraints.Size;
+import java.time.LocalDateTime;
+
+@Entity
+@Table(name = "supplier")
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@ToString
+public class DomainSupplierEntity {
```

Comment:
add new properties to the DomainSupplierEntity:
1- due date for the supplier paiment
2- company name

propage the changement to all the dto of the service.